README.md

```diff
@@ -1 +1,56 @@
-# Module1-Capstone-Web-development-course
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Module 1 Capstone
+
+> Microverse program
+
+![screenshot](./app_screenshot.png)
+
+
```

Comment:
- [ ] Remember, you need to add a [video presentation](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Once%20you%20have,pull%20request%20description.) on your project. Please do it. 🐶 
- The video presentation should be on the pull request description, but why not add it here too? 😄